### DISCUSSION TITLE
Is it possible to use Plausible from file://?

### DISCUSSION THREAD

**Discussion by TAINCER** [plausible/analytics#1161] [Anything Else]
For instance, when tracking events in an Electron app, the page is loaded with the file:// protocol.

Is this possible? When I try to run it from there I get an HTTP Error 400 Code back.

**Comment by metmarkosaric** [comment]
I know that some people "hack" our API and send data to it when tracking mobile apps so this may be something that's possible for you to figure out too but we don't support this officially so don't have any documentation on it at this point.

**Comment by ukutaht** [comment]
HTTP 400 sounds weird, that shouldn't happen. I don't have an electron setup locally but could you send me the request body from the network tab in your electron devtools? Then I could try to reproduce it locally.
